src/components/CoursePage/LevelNavigation.js

```diff
@@ -13,39 +13,41 @@ import CollapsiblePanel from '../CollapsiblePanel';
 import scrollToComponent from 'react-scroll-to-component';
 import {lessonListId} from './LessonList';
 
-const LevelNavigation = ({t, levels, isStudentMode}) => {
-  const levelListItems = levels.map(level => {
-    const onClick=() => scrollToComponent(document.getElementById(lessonListId(level)), {align: 'top'});
-    return (
-      <ListGroupItem key={level} {...{onClick}}>
+class LevelNavigation extends React.PureComponent {
+  handleClick = level => scrollToComponent(document.getElementById(lessonListId(level)), {align: 'top'});
+
+  render() {
+    const {t, levels, isStudentMode} = this.props;
+    const levelListItems = levels.map(level => (
+      <ListGroupItem key={level} onClick={() => this.handleClick(level)}>
         <span className={styles.name}>
           <LevelIcon {...{level}}/>{t('general.levels.' + level)}
         </span>
       </ListGroupItem>
```

Comment:
Hvis du tester følgende kode, så ser du at du kan løse dette på ihvertfall to måter:
```
class LevelNavigation extends React.PureComponent {
  //handleClick = level => scrollToComponent(document.getElementById(lessonListId(level)), {align: 'top'});

  showClick = (e) => {
    console.log('e.target:', e.target);
    console.log('e.target.dataset:', e.target.dataset);
    console.log('e.target.dataset.level:', e.target.dataset.level);

    console.log('e.currentTarget:', e.currentTarget);
    console.log('e.currentTarget.dataset:', e.currentTarget.dataset);
    console.log('e.currentTarget.dataset.levelCurrent:', e.currentTarget.dataset.levelCurrent);
  };

  render() {
    const {t, levels, isStudentMode} = this.props;
    const levelListItems = levels.map(level => (
      <ListGroupItem key={level} data-level-current={level} onClick={this.showClick}>
        <span data-level={level} className={styles.name}>
          <LevelIcon {...{level}}/>{t('general.levels.' + level)}
        </span>
      </ListGroupItem>
    ));
...
```

Jeg foreslår kanskje
```
class LevelNavigation extends React.PureComponent {
  handleClick = (e) => scrollToComponent(document.getElementById(lessonListId(e.target.dataset.level)), {align: 'top'});

  render() {
    const {t, levels, isStudentMode} = this.props;
    const levelListItems = levels.map(level => (
      <ListGroupItem key={level} onClick={this.handleClick}>
        <span data-level={level} className={styles.name}>
          <LevelIcon {...{level}}/>{t('general.levels.' + level)}
        </span>
      </ListGroupItem>
    ));
...
```

Reply:
Valgte denne ja. Men flyttet ````data-level={level}``` opp til ListGroupItem fordi jeg vil trykke på hele boksen og ikke bare teksten